submissions/AndriiZahumnyi/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>HTML & CSS practice: Hooli-style Popup</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="main_style.css">
+</head>
+<body>
+  <header class="header">
+    <nav class="head_nav">
+      <div class="container">
+        <ul class="main_menu">
+          <li>Mail</li>
+          <li>Images</li>
+          <li>
+            <input id="check01" type="checkbox" name="menu" />
+            <label for="check01"><img class="popup_png" src="/popup-icons/popup-button.png" alt="popup button"
+                tabindex="1"></label>
+            <div class="sub_menu">
+              <div class="sub_container">
+                <a href="#">
+                  <div class="align_self" tabindex="1">
+                    <img src="/popup-icons/calendar.png" alt="calendar icon">
+                    <p>Calendar</p>
+                  </div>
```

Comment:
You can't add block tag `div` inside inline-element `a`.
If you want turn on focus on your page you should style this in your css file with help pseudo-class `:focus`.
I think that the last word in attribute `alt` isn't needed in this case. Just "Calendar" and in the feature cases the same example.